intellij/src/saros/intellij/ui/wizards/pages/moduleselection/ModuleTab.java

```diff
@@ -408,6 +481,93 @@ private boolean hasValidExistingModule() {
     return selectedExistingModule != null && selectedExistingModule.getName().equals(moduleName);
   }
 
+  /**
+   * Updates whether the field for the new module name is marked as invalid. The new state is based
+   * on the returned value of {@link #hasValidNewModuleName()}.
+   */
+  private void updateNewModuleNameValidityIndicator() {
```

Comment:
This code duplication does not hurt, but we can think about a better design. This might be a good candidate for a Saros-eat-your-own-dog-food-session :relaxed: